src/components/stats/PlayersGameStatsList.js

```diff
@@ -1,46 +1,65 @@
 import { Fragment } from "react";
+import {
+  formatAndSortPlayersStats,
+  createPlayerNamesArray,
+  createCategeriesArray,
+} from "../../services/playersGameStatsService";
 import stats from "../../store/stats";
 import classes from "./PlayersGameStatsList.module.css";
 
 const PlayersGameStatsList = (props) => {
-  const teamStats = props.chosenTeam.slice().sort((playerA, playerB) => {
-    return playerB.pts > playerA.pts ? 1 : playerA.pts > +playerB.pts ? -1 : 0;
-  });
-
-  console.log(teamStats);
-  const playersNames = teamStats.map((player) => player.name);
-  console.log(playersNames);
-  const formattedstats = teamStats.map((player, index) => {
-    delete player.name;
-    player = Object.entries(player);
-    const name = player.unshift(["name", playersNames[index]]);
-    return player;
-  });
-
-  const categoriesArray = formattedstats[1].map(
-    (category) => (category = category[0])
-  );
+  const formattedPlayersStats = formatAndSortPlayersStats(props.chosenTeam);
```

Comment:
I wonder if instead passing the data as it is returned from the API to the component, and then using helpers functions to manipulate it, you should have created NbaStatsService that would abstract it all. Meaning it would get the data from the API, manipulate it, and then return data that is ready to use except of HTML tags and styling.
There's no right or wrong here, just a matter of making the code more readable and arranged